Steps4Impact/LoginV2/LoginV2ViewController.swift

```diff
@@ -330,3 +351,9 @@ extension LoginV2ViewController: ASAuthorizationControllerDelegate, ASAuthorizat
     AppController.shared.present(alert: alert, in: self, completion: nil)
   }
 }
+
+
+extension Foundation.Notification.Name {
+  static let googleSignInSuccess = Foundation.Notification.Name(rawValue: "googleSignInSuccess")
+  static let googleSigninFailure = Foundation.Notification.Name(rawValue: "googleSigninFailure")
```

Comment:
I think that we should be explicit here about the extension being private.  Extensions on unowned types is something to be avoided.

```suggestion
  internal static let googleSignInSuccess = Foundation.Notification.Name(rawValue: "googleSignInSuccess")
  internal static let googleSigninFailure = Foundation.Notification.Name(rawValue: "googleSigninFailure")
  internal static let firebaseFailure = Foundation.Notification.Name(rawValue: "firebaseFailure")
```

Reply:
Thanks, good to know, updated!